DQMOffline/L1Trigger/src/L1TMuonDQMOffline.cc

```diff
@@ -11,7 +11,7 @@
 #include "DataFormats/L1TMuon/interface/RegionalMuonCandFwd.h"
 #include "DataFormats/L1Trigger/interface/Muon.h"
 #include "DQMServices/Core/interface/DQMStore.h"
-#include "DataFormats/Histograms/interface/MEtoEDMFormat.h"
+//#include "DataFormats/Histograms/interface/MEtoEDMFormat.h"
```

Comment:
Please remove commented out code, or add another line of comment which explains why it must be kept.